### DISCUSSION TITLE
Using Signet to Learn the Ropes

### DISCUSSION THREAD

**Discussion by PutridQaote** [ordinals/ord#1543] [Q&A] [answered]
I'm working on getting up and running inscribing, and I've heard Casey suggest doing stuff on signet before doing inscriptions on mainnet, but I don't see anything in the docs about signet. Any guidance on how to make inscriptions on signet?

**Comment by raphjaph** [comment, marked as answer]
just add the `--signet` flag after `ord` to use signet. Start `bitcoind` with `-signet -txindex`. You can look at the docs for commands by just passing the `--help` flag
